test/unit/presenters/document_export_presenter_test.rb

```diff
@@ -221,22 +221,38 @@ class DocumentExportPresenterTest < ActiveSupport::TestCase
   end
 
   test "includes translations" do
-    edition = create(:news_article)
-    french_translation = edition.translations.create!(
-      locale: "fr", title: "Bonjour", summary: "Ça va", body: "ça va bien merci",
-    )
+    edition = create(:news_article,
+                     title: "Hello",
+                     summary: "Are you well?",
+                     body: "I am well, thank you")
+
+    english_translation = edition.translations.first
+    french_translation = edition.translations
+                                .create!(locale: "fr",
+                                         title: "Bonjour",
+                                         summary: "Ça va",
+                                         body: "ça va bien merci")
 
     result = DocumentExportPresenter.new(edition.document).as_json
-    expected = { id: french_translation.id,
-                 locale: :fr,
-                 title: "Bonjour",
-                 summary: "Ça va",
-                 body: "ça va bien merci",
-                 created_at: french_translation.created_at,
-                 updated_at: french_translation.updated_at,
-                 base_path: "/government/news/news-title.fr" }
-    assert_equal expected,
-                 result.dig(:editions, 0, :translations, 1)
+    expected = [
+      { id: english_translation.id,
+        locale: :en,
```

Comment:
Yes, odd that this comes out as a symbol! Have looked at the presenter and can't see an obvious reason for it.
But not an issue as it comes out as a JSON string in the end anyway.